packages/table/src/utils/spinner.spec.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
```

Comment:
are these tests also copied from synchro-charts?

Reply:
Yes. These tests also copied from sychro-charts with some modifications to make it work with React.